### PR TITLE
Adding SqlServer connector option to UI

### PR DESCRIPTION
**Pull request by mdrillin**
Adds the SqlServer connector type to the UI.
- Adds ConnectorIntegrator for SqlServer similar to other connectors
- Adds sqlServer instance to docker-compose
- Adds formatting for SqlServer properties in the UI

### CODE REVIEW

**Comment by indraraj** [comment]
@mdrillin is this pr good to review?

File: .github/workflows/backend-testing.yml [hunk 1]
```diff
@@ -56,4 +56,4 @@ jobs:
       - name: Build debezium-checkstyle
         run: mvn clean install -f debezium/pom.xml -pl support/checkstyle -DskipTests -DskipITs
       - name: Test Debezium UI Backend
-        run: mvn clean install -f dbzui -B -Ddebezium.test.records.waittime=20 -Dformat.formatter.goal=validate -Dformat.imports.goal=check -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        run: mvn clean install -f dbzui -B -Ddebezium.test.records.waittime=40 -Dformat.formatter.goal=validate -Dformat.imports.goal=check -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```
Comment (rk3rn3r) [inline .github/workflows/backend-testing.yml:59]: This can be reverted. Higher timeouts will lead to builds stopping during build.

Comment (mdrillin) [reply .github/workflows/backend-testing.yml:59]: Yeah, I will revert.  It was my attempt to get the backend test to complete.

**Comment by rk3rn3r** [comment]
LGTM for the backend part :heavy_check_mark:

**Comment by mdrillin** [comment]
@indraraj  - yes, it can be reviewed.  Just a matter of if we want to merge it with the backend test failing...  btw, here are the properties I've been using to connect to the sqlServer test instance

Namespace:  fullfillment
Hostname:   dbzui-db-sqlserver
User:       sa
Password:   Password!
Database:   testDB
Instance:   dbzui-db-sqlserver
Kafka broker: kafka:9092
topic name: dbhistory.fullfillment

**Review by uidoyen — APPROVED** [review]
Tested locally, able to create SQL connector. Cypress test is passing locally too.

**Comment by indraraj** [comment]
> @indraraj - yes, it can be reviewed. Just a matter of if we want to merge it with the backend test failing... btw, here are the properties I've been using to connect to the sqlServer test instance
> 
> Namespace: fullfillment Hostname: dbzui-db-sqlserver User: sa Password: Password! Database: testDB Instance: dbzui-db-sqlserver Kafka broker: kafka:9092 topic name: dbhistory.fullfillment

thanks for sharing this

**Comment by mdrillin** [comment]
I am merging this one.  The tests run fine locally - the backend issues still need to be resolved at some point.

**Pull request merged**